unit/solvers/refinement/string_constraint_generator_valueof/is_digit_with_radix_lower_case.cpp

```diff
@@ -15,43 +15,99 @@
 #include <util/simplify_expr.h>
 #include <util/std_types.h>
 
-SCENARIO("is_digit_with_radix_lower_case",
-  "[core][solvers][refinement][string_constraint_generator_valueof]")
+static exprt actual(const mp_integer int_value, const unsigned long radix_ul)
```

Comment:
Could we have a one line description of what that function does? Same for `actual_with_radix`.